--- ../RustGnuplot/doc/gnuplot/main.js	2013-11-02 22:48:23.873140463 -0400
+++ doc/gnuplot/main.js	2013-10-31 15:47:19.394478847 -0400
@@ -16,6 +16,9 @@
     var resizeTimeout, interval;
 
     $('.js-only').removeClass('js-only');
+    
+    /* SIEGE PATCH */
+    $('.search-input').attr('placeholder', 'Search documentation...');
 
     function resizeShortBlocks() {
         if (resizeTimeout) {
